src/c++/tests/memory_leak_test.cc

```diff
@@ -105,70 +105,93 @@ ValidateResult(
   std::cout << result->DebugString() << std::endl;
 }
 
+void
+ValidateResponse(
+    std::shared_ptr<tc::InferResult> results_ptr,
```

Comment:
can we make this const as well to  keep it consistent with validate result?

Reply:
Done